### PR TITLE
Update CLI project example DAGs

### PR DESCRIPTION
**Pull request by kentdanas**
## Description

Update the example DAGs to be consistent with what we're using in trial flow and our Learn tutorials, as discussed with @paolaperaza 

## 🎟 Issue(s)

Related #XXX

## 🧪 Functional Testing

> List the functional testing steps to confirm this feature or fix.

## 📸 Screenshots

> Add screenshots to illustrate the validity of these changes.

## 📋 Checklist

- [ ] Rebased from the main (or release if patching) branch (before testing)
- [ ] Ran `make test` before taking out of draft
- [ ] Ran `make lint` before taking out of draft
- [ ] Added/updated applicable tests
- [ ] Tested against [Astro-API](https://github.com/astronomer/astro/) (if necessary).
- [ ] Tested against [Houston-API](https://github.com/astronomer/houston-api/) and [Astronomer](https://github.com/astronomer/astronomer/) (if necessary).
- [ ] Communicated to/tagged owners of respective clients potentially impacted by these changes.
- [ ] Updated any related [documentation](https://github.com/astronomer/docs/)

### CODE REVIEW

File: airflow/airflow.go [hunk 1]
```diff
@@ -13,13 +13,10 @@ import (
...
 var (
-	//go:embed include/advancedexampledag.py
-	ExampleDagAdvanced string
-
 	//go:embed include/astronomermonitoringdag.py
 	MonitoringDag string
 
-	//go:embed include/basicexampledag.py
+	//go:embed include/exampledag.py
 	ExampleDagBasic string
```
Comment (kentdanas) [inline airflow/airflow.go:20]:
```suggestion
	ExampleDag string
```

**Comment by sunkickr** [comment]
Were you able to test this DAG against `astro dev parse/pytest`. Just want to make sure all the tests pass out of the box. If we need to change the tests to make it so that is okay

**Comment by kentdanas** [comment]
> Were you able to test this DAG against `astro dev parse/pytest`. Just want to make sure all the tests pass out of the box. If we need to change the tests to make it so that is okay

@sunkickr yes, the new DAG passes all the tests!

**Review by kushalmalani — APPROVED** [review]
LGTM

**Comment by kentdanas** [comment]
Thanks @kushalmalani! Are you able to merge this? I'm not authorized

**Pull request merged**